pkg/master/controller.go

```diff
@@ -263,6 +264,11 @@ func (c *Controller) CreateOrUpdateMasterServiceIfNeeded(serviceName string, ser
 			ClusterIP:       serviceIP.String(),
 			SessionAffinity: api.ServiceAffinityClientIP,
 			Type:            serviceType,
+			SessionAffinityConfig: &api.SessionAffinityConfig{
```

Comment:
Mostly was asking to understand.  In real runs, the defaulting logic WILL apply.  Tests might not go through that, and this is not injected (created internally), so OK.